src/imitation/scripts/train_adversarial.py

```diff
@@ -169,6 +165,9 @@ def train(_run,
         visualizer.add_data_disc_loss(False, epoch)
 
       trainer.train_gen(n_gen_steps_per_epoch)
```

Comment:
[Answered here](https://github.com/HumanCompatibleAI/imitation/pull/134#discussion_r362671284). TLDR because there isn't a callback inside `train()` and in fact we would need two different callbacks if we wanted to add `visualizer` data the same way that we do above.

Reply:
train was removed in #134